lnbits/core/crud.py

```diff
@@ -451,6 +451,17 @@ async def delete_wallet(
     )
 
 
+async def delete_unused_wallets(conn: Optional[Connection] = None) -> None:
+    await (conn or db).execute(
+        """
+        DELETE FROM wallets
```

Comment:
There should be an age here (older than one month for example), otherwise wallets created 1 minute ago will be deleted .

Reply:
This can be updated now: https://github.com/lnbits/lnbits/pull/2177